### ISSUE TITLE
add support for COLR/CPAL to make color-variable font

### ISSUE THREAD

**Issue by aminabedi68** [googlefonts/fontmake#673]
Hi
I need to make a color-variable font and I realized that the only supported format is COLR/CPAL(maybe I'm wrong?). I guess you are working on this and thought it's better to ask you before trying the other ways.

**Comment by anthrotype** [comment]
Only COLR/CPAL color fonts are supported by ufo2ft (and thus by fontmake which uses ufo2ft).
There's currently no documentation about this, for the time beig you'd have to figure it out by looking at the code, e.g. start from this https://github.com/googlefonts/ufo2ft/pull/359 where support for UFOs with color layers was added in ufo2ft.

The only color font format that supports variable fonts is COLR/CPAL, as it piggybacks on glyf+gvar or CFF2 variation data.
Basically if you feed fontmake a designspace and master UFOs with interpolatable outlines containing color layers in the format ufo2ft expects, then it will build a color VF.

**Comment by aminabedi68** [comment]
thank you, it's so exciting :)
